src/tqec/_cli/tqec.py

```diff
@@ -13,6 +15,7 @@ def main() -> None:
 
     Dae2ObservablesTQECSubCommand.add_subcommand(subparser)
     CheckDaeTQECSubCommand.add_subcommand(subparser)
+    Dae2CircuitsTQECSubCommand.add_subcommand(subparser)
 
-    args = parser.parse_args()
+    args = parser.parse_args(args=None if sys.argv[1:] else ["--help"])
```

Comment:
I do not get the reason for this change here. Could you explain it in more details?

Reply:
This makes `tqec` [display help message](https://stackoverflow.com/a/47440202) when no arguments are provided(call `tqec` solely from command line).